mlrun/__main__.py

```diff
@@ -251,7 +259,7 @@ def run(
 
     if func_url or kind or image:
         if func_url:
-            runtime = func_url_to_runtime(func_url)
+            runtime = func_url_to_runtime(func_url, save_project)
```

Comment:
`func_url_to_runtime` is just one of the ways to create the runtime class (from url) the flag of whether to `ensure_project` (ensure is better describing the behavior IMO) should affect all flows

Reply:
Talked offline, will leave it as save for now to keep it aligned with save function.